Automattic/newspack-theme
# Update screenshots to match demo sites

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates all the screenshots (Newspack, Joseph, Katharine, Nelson, Sacha, Scott) to match their demo sites.

### How to test the changes in this Pull Request:

1. Navigate to Appearance > Themes
2. Check the screenshots
3. Switch to this branch
4. Recheck the screenshots -- they should be up-to-date

### Other information:
